packages/syft/src/syft/service/notifier/notifier.py

```diff
@@ -161,6 +170,9 @@ class NotifierSettings(SyftObject):
     email_username: str | None = ""
     email_password: str | None = ""
 
+    email_activity: dict[str, dict[SyftVerifyKey, UserNotificationActivity]] = {}
```

Comment:
should we add migrations for NotifierSettings?

Reply:
Yup!! Thanks for notificing that. I've added them in [74f1b9d](https://github.com/OpenMined/PySyft/pull/9052/commits/74f1b9da8124b0468a2c59fca716d17e103f8916)